android/app/src/main/assets/README.md

```diff
@@ -1,14 +1,31 @@
 # Android App
-Files must be placed in the assets directory
-# Required files to run all samples:
+[**Return to aws-iot-device-sdk-java-v2 README**](../../../../../../aws-iot-device-sdk-java-v2/README.md)
+
+# Files used by IoT Samples App
+Files must be placed in the assets directory for the sample app to connect to IoT Core.
+## Required by all samples:
```

Comment:
It seems that `certificate` and `privatekey` are not required for Cognito sample, could we also make it clearer here?